nautobot/extras/datasources/utils.py

```diff
@@ -29,12 +29,9 @@ def files_from_contenttype_directories(base_path, job_result, log_grouping):
             try:
                 model_content_type = ContentType.objects.get(app_label=app_label, model=modelname)
             except ContentType.DoesNotExist:
-                job_result.log(
-                    f"Skipping `{app_label}.{modelname}` as it isn't a known content type",
-                    level_choice=LogLevelChoices.LOG_WARNING,
-                    grouping=log_grouping,
-                    logger=logger,
-                )
+                msg = f"Skipping `{app_label}.{modelname}` as it isn't a known content type"
+                logger.warning(msg)
+                job_result.log(msg, level_choice=LogLevelChoices.LOG_WARNING, grouping=log_grouping)
```

Comment:
Is this redundant?

Reply:
The first line logs to the console, the second line logs to the database as a `JobLogEntry`